File: JSBASIC/157_array_copy.js
// 157	157_array_copy.js	Copy one array into another array.

let original = [10, 20, 30, 40, 50];
let copy = [...original];
console.log("Original:", original);
console.log("Copied array:", copy);
copy[0] = 99;
console.log("After modifying copy, original unchanged:", original);

File: JavaBasic/157_array_copy.java
// 157	157_array_copy.java	Copy one array into another array.
import java.util.Arrays;
public class _157_array_copy {
    public static void main(String[] args) {
        int[] original = {10, 20, 30, 40, 50};
        int[] copy = Arrays.copyOf(original, original.length);
        System.out.println("Original: " + Arrays.toString(original));
        System.out.println("Copy: " + Arrays.toString(copy));
    }
}
